redbot/cogs/mod/kickban.py

```diff
@@ -179,6 +179,20 @@ class KickBanMixin(MixinMeta):
             return
         audit_reason = get_audit_reason(author, reason)
         try:
+            if reason == None:
+                reason = "No reason was given."
+            else:
+                reason = reason
+            self == self.bot.get_cog("Mod")
```

Comment:
Is this supposed to be a `=`?  And if so, `self` is already the Mod cog, so this wouldn't be needed.